packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -1,23 +1,31 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import ContractType from './contract-type.jsx';
-import { getContractTypeCategoryIcons, findContractCategory } from '../../../Helpers/contract-type';
+import ContractType from './contract-type';
+import { getContractTypeCategoryIcons, findContractCategory } from '../../../Helpers/contract-type.js';
+import { TList, TContractType, TContractCategory } from './ContractTypeInfo/contract-type-info.js';
+
+type TContractTypeWidget = {
+    name?: string;
+    value: TContractType['value'];
+    list: TContractCategory[];
+    onChange?: (event: DeepPartial<React.ChangeEvent<HTMLInputElement>>) => void;
```

Comment:
```suggestion
    onChange: (event: DeepPartial<React.ChangeEvent<HTMLInputElement>>) => void;
```
@henry-deriv nit-pick: onChange is always passed here, ain't it? :)

Reply:
sure